app/classifier/tasks/survey/choice.cjsx

```diff
@@ -44,15 +44,19 @@ ImageFlipper = React.createClass
 module.exports = React.createClass
   displayName: 'Choice'
 
+  propTypes:
+    annotation: React.PropTypes.object.isRequired
```

Comment:
I thought you added this prop? Renaming it is an excellent idea if the name is confusing.

Re. confusing names, `choice` and `choiceID` being used for the same thing in `Index` always confuses me, but `choice` is baked into the annotations now, so renaming it would break aggregation for the survey task 😞